themes-default/slim/webpack.config.js

```diff
@@ -15,17 +17,16 @@ const { cssThemes } = pkg.config;
 const perTheme = action => Object.values(cssThemes).map(theme => action(theme));
 
 /**
- * Helper function to simplify CopyWebpackPlugin configuration when copying assets from `./dist`.
+ * Helper function to simplify FileManagerPlugin configuration when copying assets from `./dist`.
  * To be used in-conjunction-with `perTheme`.
  * @param {string} type - Asset type (e.g. `js`, `css`, `fonts`). Must be the same as the folder name in `./dist`.
  * @param {string} [search] - Glob-like string to match files. (default: `**`)
- * @returns {function} - A function the recieves the theme object from `perTheme`.
+ * @returns {function} - A function the receives the theme object from `perTheme`.
```

Comment:
"that receives"

Reply:
Oh man the typos :| Let's just ignore "the" (that) one for now